vendor/k8s.io/system-validators/validators/types_unix.go

```diff
@@ -73,7 +73,7 @@ var DefaultSysSpec = SysSpec{
 	RuntimeSpec: RuntimeSpec{
 		DockerSpec: &DockerSpec{
 			Version:     []string{`1\.1[1-3]\..*`, `17\.0[3,6,9]\..*`, `18\.0[6,9]\..*`, `19\.03\..*`, `20\.10\..*`},
-			GraphDriver: []string{"aufs", "overlay", "overlay2", "devicemapper", "zfs"},
+			GraphDriver: []string{"aufs", "btrfs", "overlay", "overlay2", "devicemapper", "zfs"},
```

Comment:
unrelated to this PR / need to follow up somewhere else but uh huh, are any of these tested?

Reply:
good question

Reply:
sig node must know.
it seems cadvasior vendor bump have historically fixed "issue X with graph driver foo in CR bar".